app/views/layouts/_navigation.html.erb

```diff
@@ -0,0 +1,26 @@
+
+
+<nav class="bg-white border-gray-200 dark:bg-gray-900">
+  <div class="max-w-screen-xl flex flex-wrap items-center justify-between mx-auto p-4">
+    <div class="flex items-center space-x-3 rtl:space-x-reverse">
+        <%= link_to image_tag("One_piece_logo.png"), recipes_path, class: "h-8 mt-1 logo-nav" %>
+        <span class="self-center text-2xl font-semibold whitespace-nowrap dark:text-white">Mame-Zefu</span>
```

Comment:
<img width="161" alt="image" src="https://github.com/CodeSmitty/mame-zefu/assets/3910847/e22d26d9-c7c3-4287-9729-239b3cbac703">

I feel like the text should be either as tall as the logo or centered vertically and maybe closer to the logo.  It would also be nice if it was part of the link. Not sure how I feel about the hyphen, maybe try out 'Mame Zefu' and 'MameZefu' and decide which of the 3 you like better.

Reply:
Yea, I can mess with the alignment, it will be better once I get the logo. I was probably do that tonight, and have a logo made in the next couple of days. I honestly like 'MameZefu' the most.